.github/workflows/README.md

```
@@ -50,25 +50,37 @@ Approval & Auto-Merge
 - $1/$5 per million tokens (5x cheaper than Opus 4.5)
 - 2x faster than Sonnet
 
-#### 2. Claude Auto-Heal (`.github/workflows/claude-autoheal.yml`)
-**Continuous healing** when AI agents post feedback.
+#### 2. Claude Equalizer (`.github/workflows/equalizer.yml`)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)